custom_model_runner/datarobot_drum/drum/main.py

```diff
@@ -115,6 +115,10 @@ def _is_arg_registered_in_subparser(sub_parser, arg):
             if sub_parser is None:
                 return
 
+            skip_predict = os.environ.get("SKIP_PREDICT")
+            if skip_predict is not None:
+                sys.argv.extend([ArgumentsOptions.SKIP_PREDICT])
+
             for env_var_key in ArgumentOptionsEnvVars.VALUE_VARS + ArgumentOptionsEnvVars.BOOL_VARS:
```

Comment:
Why don't you use existing boilerplate?
Add SKIP_PREDICT into ArgumentOptionsEnvVars.BOOL_VARS ...

Reply:
sure